### ISSUE TITLE
globalpointer_loss的疑惑

### ISSUE THREAD

**Issue by hquzhuguofeng** [JunnYu/GPLinker_pytorch#11]
`
def globalpointer_loss(y_pred, y_true):
    shape = y_pred.shape
    # bs, nclass, max_spo_num
    y_true = y_true[..., 0] * shape[2] + y_true[..., 1]
    # bs, nclass, seqlen * seqlen
    y_pred = y_pred.reshape(shape[0], -1, np.prod(shape[2:]))
    loss = sparse_multilabel_categorical_crossentropy(
        y_true, y_pred, mask_zero=True)
    return loss.sum(dim=1).mean()
`

请问下大佬y_true[..., 0] * shape[2] 为什么要乘shape[2]，这样y_true不就变化了吗？这里我不太明白~

**Comment by hquzhuguofeng** [comment]
2维度bian
